### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by artemrizhov**

### CODE REVIEW

File: hanging_threads.py [hunk 1]
```diff
@@ -19,13 +19,6 @@
 """
 import sys
 import threading
-try:
-    try:
-        from threading import _get_ident as get_ident
-    except ImportError:
-        from threading import get_ident
-except ImportError:
-    from thread import get_ident
```
Comment (niccokunzmann) [inline hanging_threads.py:28]: looks like we do not need this because we use threading. current_thread, right? :)

Comment (artemrizhov) [reply hanging_threads.py:28]: @niccokunzmann Yes, exactly.

**Comment by artemrizhov** [comment]
@niccokunzmann Strange thing. Last merges created new commits for each commit from my PR, and my branches are shown as not merged copies. Looks like the branches have been rebased. Do you do something special for this?

**Comment by artemrizhov** [comment]
@niccokunzmann For example you can compare the commit hashes in my branch and in the master branch. They all differ.
https://github.com/artemrizhov/hanging_threads/commits/cleanup
https://github.com/niccokunzmann/hanging_threads/commits/master

**Comment by niccokunzmann** [comment]
@artemrizhov Uh, yes, I rebased to try it out. Should I rather create a merge commit?

**Comment by artemrizhov** [comment]
@niccokunzmann Yes, from future PRs please. :)

**Pull request merged**